tool/settings.re

```diff
@@ -150,6 +152,8 @@ impl Settings {
             player_color_red: get_float("player_color_red", 0.),
             player_color_green: get_float("player_color_green", 0.),
             player_color_blue: get_float("player_color_blue", 0.),
+            flying_up_down_vel: get_float("flying_up_down_vel", 300.),
+            flying_forward_backward_vel: get_float("flying_forward_backward_vel", 600.),
```

Comment:
Are those sane defaults? When I tested, I thought 300 up/down felt pretty slow.

Reply:
The latter default is the default in UE.